src/test/java/com/datasonnet/HeaderTest.java

```diff
@@ -141,4 +142,28 @@ void testUnterminatedHeaderFailsNicely() {
                     "version=2.0\n");
         });
     }
+
+    @Test
+    public void test_default_output() throws HeaderParseException {
+        Header header1 = Header.parseHeader("/** DataSonnet\n" +
+                "version=2.0\n" +
+                "output application/x-java-object;q=0.9\n" +
+                "output application/json;q=1.0\n" +
+                "*/");
+
+        assertTrue(header1.getDefaultOutput().isPresent());
+        assertTrue(MediaTypes.APPLICATION_JSON.equalsTypeAndSubtype(header1.getDefaultOutput().get()));
+    }
+
+    @Test
+    public void test_default_input() throws HeaderParseException {
+        Header header1 = Header.parseHeader("/** DataSonnet\n" +
+                "version=2.0\n" +
+                "input payload application/json;q=0.9\n" +
+                "input payload application/x-java-object;q=1.0\n" +
```

Comment:
In both of these, the second header is the default one. Could we modify one of the tests so in it the first header is the right one, to eliminate the possibility of code that just picks the second header passing?

Reply:
haha sure. I actually moved them around while testing for that exact reason

Reply:
as in, it's funny because I thought of the same exact thing but then forgot to leave them in different order in the tests